spec/unit/controllers/services/service_instances_controller_spec.rb

```diff
@@ -4026,20 +4054,13 @@ def verify_forbidden(user)
 
               it "has a #{expected_status} http status code" do
                 get "/v2/service_instances/#{instance.guid}/shared_to"
-                expect(last_response.status).to eq(expected_status), "Expected #{expected_status}, got: #{last_response.status}, role: #{role}"
+                expect(last_response.status).to eq(expected_status), "Expected #{expected_status}, got: #{last_response.status}, role: #{role}, response: #{last_response.body}"
```

Comment:
Is the `last_response.body` part intentional? We often use this during development for extra information, but typically don't commit uses of it.

Reply:
This was probably unintentional! We will remove it.